src/ephemeron/dune

```diff
@@ -0,0 +1,28 @@
+;; Linearizability tests of the stdlib Ephemeron module
+
+;; this prevents the tests from running on a default build
+(alias
+ (name default)
+ (deps lin_tests.exe))
+
+(env
+ (_
+  (binaries
+   (../check_error_count.exe as check_error_count))))
+
+
+(executable
+ (name lin_tests)
+ (modules lin_tests)
+ (flags (:standard -w -27))
+ (libraries util qcheck lin)
+ (preprocess (pps ppx_deriving_qcheck ppx_deriving.show)))
+
+(rule
+ (alias runtest)
+ (action
+  (progn
+   (with-accepted-exit-codes 1
+    (with-stdout-to "lin-output.txt" (run ./lin_tests.exe --no-colors --verbose)))
+   (cat lin-output.txt)
+    (run %{bin:check_error_count} "lin_tests" 1 lin-output.txt))))
```

Comment:
Why do you expect 1 error precisely?
Are ephemeron's not domain-safe - or is it failing for another reason?

Reply:
Ephemeron are failing the test with domains. But it shouldn't.
Expected behaviour is to have `0` here. I'll correct it.